apps/codeforafrica/src/pages/404.page.js

```diff
@@ -1,12 +1,12 @@
 import ErrorPage from "@/codeforafrica/components/ErrorPage";
-import { getPageStaticProps } from "@/codeforafrica/lib";
+import { getPageStaticProps } from "@/codeforafrica/lib/data/rest";
 
 function NotFound(props) {
   return <ErrorPage {...props} />;
 }
 
-export async function getStaticProps() {
-  return getPageStaticProps({ slug: `/404` });
+export async function getStaticProps(context) {
+  return getPageStaticProps(context, "/404");
```

Comment:
🤔 Why are we passing `slug` separate from `context`? I think `slug` is just [`params`](https://nextjs.org/docs/pages/api-reference/functions/get-static-props#context-parameter), no?

Reply:
@kilemensi I think we're going to do away with this. We have an issue for [Error Pages](https://github.com/CodeForAfrica/ui/issues/565)